docs/tables/snowflake_resource_monitor.md

```diff
@@ -0,0 +1,45 @@
+# Table: snowflake_warehouse
+
+A warehouse is a cluster of compute resources in Snowflake. Warehouses provide the required resources, such as CPU, memory, and temporary storage, to perform queries.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  credit_quota,
+  used_credits,
+  remaining_credits
+from
+  snowflake_resource_monitor;
+```
+
+### List warehouses and % credit left
+
+```sql
+select account as "Account",
+    name as "Warehouse",
+    credit_quota as "Credit Quota",
+    used_credits as "Used Credits",
+    remaining_credits as "Remaining Credits",
+    to_char(used_credits/credit_quota*100,'9990D9') as "% Used",
+    case
+        when used_credits/credit_quota*100 > 90 then 'alert'
+        when used_credits/credit_quota*100 > 75 then 'warning'
+        else 'ok'
+    end as type
+from snowflake_resource_monitor
+where used_credits/credit_quota*100 > 75
+order by used_credits/credit_quota desc
+```
+
+### List warehouses which have used all their credits
+
+```sql
+select account as "Account",
+    name as "Warehouse"
+from snowflake_resource_monitor
+where remaining_credits < 1
```

Comment:
```suggestion
select
  account,
  name as warehouse
from
  snowflake_resource_monitor
where
  remaining_credits < 1;
```

We don't usually convert column names to use mixed case (as Postgres treats identifiers as lowercase), so `account` should remain as is, but showing `name` as `warehouse` is helpful.

Please refer to some existing Snowflake table docs we have for more examples, e.g., https://hub.steampipe.io/plugins/turbot/snowflake/tables/snowflake_schemata.

Reply:
remove Account and changed warehouse